Formula/jdep.rb

```diff
@@ -0,0 +1,104 @@
+class Jdep < Formula
+  desc "Analyze Java .class files and produce Makefile dependencies."
+  homepage "http://www.fudco.com/software/jdep.html"
+  url "http://www.fudco.com/software/jdep-1.3.tar.gz"
+  sha256 "3dd925ae2523cca59e4aaa7cbb8daa28927b40520f0f8dccead91d9edea1dc65"
+
+  patch :DATA
```

Comment:
Could you replace this with a patch do block, along with a comment linking to the upstream PR?
```ruby
patch do
  url "https://github.com/FUDCo/jdep/commit/452bd87c69a1c0f03f381b32e995b0b9a813fc25.diff"
  sha256 "55c7be39f34ed974e4ed2213626ded6205df07dc9e8099e2f49f63118540a993"
end
```
(Inline patches can get mangled easily.)